debian/control

```diff
@@ -0,0 +1,24 @@
+Source: libjsonv
+Priority: optional
+Maintainer: Remy Chibois <chybz@chybz.net>
+Build-Depends: debhelper (>= 9), cmake
+Standards-Version: 3.9.5
+Section: libs
+Homepage: https://github.com/tgockel/jsonv
```

Comment:
I've normally been treating the [documentation](http://tgockel.github.io/json-voorhees/) as the "official" homepage, but I don't know if that's the "normal" thing to do :smiley_cat: